### ISSUE TITLE
Is it possible to query a local working copy of the ontology?

### ISSUE THREAD

**Issue by bbartley** [Bioprotocols/container-ontology#17]
I haven't totally deciphered the Makefile, but I'm guessing queries are directed at a web server somewhere?  In any event, my local changes to the ontology file aren't returned by queries.  Is there a way to query the local turtle file?

**Comment by rpgoldman** [comment]
No, I'm afraid not, because we want information from the closure of the Turtle file that is computed by the server.

It would be possible to precompute and store it, but that's not done now.

Doing `make server` should give you a running server and make the queries go through.

I don't have any great ideas about how to ensure that the container server is started up when needed; if you do, please let me know. (e.g., the container server might be remote, so we can't just always start it up when we load the container API)
